src/main/features/offer/views/offer-confirmation.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, submitButton %}
+{% from "support.njk" import supportWidget %}
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+        <div class="govuk-box-highlight">
+          <h1 class="bold-large">{{ t('Your offer has been sent to the %s',
```

Comment:
> Your offer has been sent to the Jane Smith
That content doesn't look right with *t* near the end

Reply:
yes you are right, it doesn't look right, this is from prototype, as it is, nothing added, nothing removed

Reply:
do you think that makes it right?

Reply:
no, but as a developer you believe prototype is always right, because that is what is presented and agreed by the client. What do you think?

Reply:
Mistakes happen, UX team can make them as well. If your gut tells you that something is not right it's always best to double-check.